app/src/main/java/io/homeassistant/companion/android/widgets/media_player_controls/MediaPlayerControlsWidget.kt

```diff
@@ -231,15 +230,16 @@ class MediaPlayerControlsWidget : BaseWidgetProvider() {
                     )
                 }
 
-                if (icon == null || !icon.startsWith("mdi") || !icon.contains(":")) {
-                    icon = "mdi:cast"
+                var iconBitmap = IconicsDrawable(context, "cmd-cast").toBitmap()
```

Comment:
Minor performance improvement: the icon is known, so instead of using the string `"cmd-cast"` which will search all icons you can just specify `CommunityMaterial.Icon.cmd_cast`.

Reply:
> Minor performance improvement: the icon is known, so instead of using the string `"cmd-cast"` which will search all icons you can just specify `CommunityMaterial.Icon.cmd_cast`.

This doesn't seem to allow me to set it `toBitmap()` which is required for `setImageViewBitmap()` which is the only method we can use thats compatible with all supported API levels.  Unless I am missing something?

Reply:
This works for me:
```suggestion
                var iconBitmap = IconicsDrawable(context, CommunityMaterial.Icon.cmd_cast).toBitmap()
```

(+ import for `com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial`)

Reply:
oh thats what you meant 😅